Jenkinsfile

```diff
@@ -30,12 +30,14 @@ pipeline {
             steps {
                 dir ('/home/jenkins/go/src/github.com/jenkins-x/jx') {
                     checkout scm
+                    sh "git config --global credential.helper store"
                     sh "jx step git credentials"
```

Comment:
I think we can remove these now that we use knative build creds init container.  Credentials get automatically mounted now so I would have thought we could remove all 
```
sh "git config --global credential.helper store"
sh "jx step git credentials"
```

Reply:
alright